apps/server/src/routers/diffRouter.ts

```diff
@@ -51,8 +50,9 @@ router.get(
 		const dict2Doc = await dictionaryService.getOneByNameAndVersion(name, rightVersion);
 		const dict1 = showReferences ? dict1Doc : replaceReferences(dict1Doc);
 		const dict2 = showReferences ? dict2Doc : replaceReferences(dict2Doc);
-		const diff = diffUtil(dict1, dict2);
-		res.status(200).send(Array.from(diff.entries()));
+		const diff = DiffUtils.getDiff(dict1, dict2);
+
+		res.status(200).send(DiffUtils.diffMapToArray(diff));
```

Comment:
Not a request for change, but a suggestion from experience, to prevent the occasional double response that's hard to troubleshoot.

```suggestion
		return res.status(200).send(DiffUtils.diffMapToArray(diff));
```